src/Shared/Input/InputField.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Input from './Input';
+import passwordShowIcon from '../../Assets/Icons/password-show.png';
+import passwordHideIcon from '../../Assets/Icons/password-hide.png';
```

Comment:
Suggestion: can import all the images form one index.js

Reply:
ok